lib/webhook-proxy.js

```diff
@@ -1,14 +1,31 @@
 module.exports = ({url, port, path, logger}) => {
   try {
+    var goodUrl = true // url is considered to be true
     const SmeeClient = require('smee-client')
-
-    const smee = new SmeeClient({
-      source: url,
-      target: `http://localhost:${port}${path}`,
-      logger
-    })
-    return smee.start()
+    if (!validURL(url)) {
+      goodUrl = false // url is invalid
+      const error = 'The provided Webhook Proxy URL is invalid.'
+      throw error
```

Comment:
This can be one line, and should throw an instance of the `Error` class:

```js
throw new Error('The provided Webhook Proxy URL is invalid.')
```

This comes with some built-in semantics like stack traces and error bubbling.

Reply:
Done. 😄